rx-java-2/src/main/java/info/metadude/kotlin/library/c3media/RxC3MediaService.kt

```diff
@@ -1,6 +1,11 @@
 package info.metadude.kotlin.library.c3media
 
-import info.metadude.kotlin.library.c3media.models.*
+import info.metadude.kotlin.library.c3media.models.Conference
+import info.metadude.kotlin.library.c3media.models.ConferencesResponse
+import info.metadude.kotlin.library.c3media.models.Event
+import info.metadude.kotlin.library.c3media.models.EventsResponse
+import info.metadude.kotlin.library.c3media.models.Recording
+import info.metadude.kotlin.library.c3media.models.RecordingsResponse
```

Comment:
Can you please check if the imports fold to `import info.metadude.kotlin.library.c3media.models.*` if you use the default IntelliJ settings. It does on my machine.